plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -337,14 +337,6 @@ public void testShowCreateTable()
                         ")\\E");
```

Comment:
> Despite a materialized view having `VIEW`
`table_type` in `information_schema.tables`, they are not supposed to
show up in `views`.

So where they should appear? Is this part of SQL standard?

Reply:
The commit message is wrong, will fix it.

we have system.metadata.materialized_views for them

Reply:
> Is this part of SQL standard?

afaik MVs are extension